adr_viewer/templates/index.html

```diff
@@ -6,6 +6,9 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
         <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+        {% if config.include_mermaid %}
+        <script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js" async></script>
```

Comment:
Can we pin this at the current major version (10.4.0) as with the other imports above/below it?

I'm going to raise an issue to make them parameters of the template so they can be upgraded (which they haven't been for a while).

Reply:
My reasoning against that is that I know a lot of people use https://mermaid.live to build their diagrams and it doesn't seem to have a way to set what version you want to use for a particular diagram. This is particularly painful when you are building your diagram in that tool in the latest version of mermaid but then you copy paste into another tool only for your diagram not to work because it is stuck on some older version. I'm actually hitting this problem right now with GitHub Enterprise being something like a year behind current mermaid version.

I guess I could add in something that lets the user specify but defaults to newest.

Maybe I could add that in a follow-up pr?

Reply:
That sounds reasonable, fair enough 👍🏻